<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralSettingsWidget</class>
 <widget class="QWidget" name="GeneralSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>General</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>10</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>10</number>
     </property>
     <property name="verticalSpacing">
      <number>15</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="langLabel">
       <property name="text">
        <string>Localization:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="langComboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="qssLabel">
       <property name="text">
        <string>Style Sheet:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>1</number>
       </property>
       <item>
        <widget class="QLineEdit" name="qssLineEdit"/>
       </item>
       <item>
        <widget class="QToolButton" name="qssToolButton">
         <property name="icon">
          <iconset resource="myimage.qrc">
           <normaloff>:/myImage/images/open-dir.png</normaloff>:/myImage/images/open-dir.png</iconset>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="autoLogCheckBox">
       <property name="text">
        <string>Auto-log</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="autoSaveCheckBox">
       <property name="text">
        <string>Auto-save</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <property name="horizontalSpacing">
          <number>20</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="autoSaveLabel">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Auto-save interval:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="autoSaveSpinBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="suffix">
            <string>min</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>60</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Default Directory</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QRadioButton" name="currentFileButton">
        <property name="text">
         <string>Current file</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="lastOperateButton">
        <property name="text">
         <string>Last operate</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="customButton">
        <property name="text">
         <string>Custom</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>1</number>
        </property>
        <item>
         <widget class="QLineEdit" name="defDirLineEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="defDirToolButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="icon">
           <iconset resource="myimage.qrc">
            <normaloff>:/myImage/images/open-dir.png</normaloff>:/myImage/images/open-dir.png</iconset>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>History Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="recentClosedButton">
          <property name="text">
           <string>Recent closed files</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="recentOpenedButton">
          <property name="text">
           <string>Recent opened files</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="recentSavedButton">
          <property name="text">
           <string>Recent saved files</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="maxHistoryLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Max history:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="maxHistoryBox"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="myimage.qrc"/>
 </resources>
 <connections/>
</ui>
